docs/source/install/sources.rst

```diff
@@ -52,63 +55,48 @@ and install required dependencies, and run tests.
 
     git clone https://github.com/StackStorm/st2.git
     cd st2
-    make all
+    make requirements
 
-Configuration
-~~~~~~~~~~~~~
-
-Specify a user for running local and remote SSH actions. In conf/st2.dev.conf, change ``ssh_key_file``
-to point to the user's key file:
-
-::
+Configure System User
+~~~~~~~~~~~~~~~~~~~~~
+Specify a user for running local and remote SSH actions. See :ref:`config-configure-ssh`. In st2/conf/st2.dev.conf, change ``ssh_key_file`` to point to the user's key file. ::
 
-    [ssh_runner]
+    [system_user]
     user = stanley
     ssh_key_file = /home/[current user]/.ssh/stanley_rsa
 
-If you don't have a user for this purpose (as in case of devenv), there are a number of steps you
-need to perform to create and setup one that works with fabric\_runner. For more information, see
-:ref:`config-configure-ssh`.
-
 Running
 ~~~~~~~
+Run the following to start |st2|. The script will start |st2| components in screen sessions. ::
 
-To run |st2| from source, it's assumed that python virtual environment
-is activated and in use.
+    sudo ./st2/tools/launchdev.sh start
```

Comment:
sudo isn't required and I believe it actually won't work if you use sudo while following this guide.

`make requirements` creates a virtual environment, activates it and installs the dependencies there. Using sudo means virtual environment which is currently activated won't be preserved so starting stuff would fail.